samples/nextjs/src/lib/layout-personalization-service-factory.ts

```diff
@@ -0,0 +1,28 @@
+import config from 'temp/config';
+import {
+  LayoutPersonalizationService,
+  RestPersonalizationDecisionsService,
+  GraphQLLayoutFragmentService,
+} from '@sitecore-jss/sitecore-jss-nextjs';
+
+export class LayoutPersonalizationServiceFactory {
+  create(): LayoutPersonalizationService {
+    return new LayoutPersonalizationService(
+      new RestPersonalizationDecisionsService({
+        serviceUrl: config.personalizationDecisionsEndpoint,
+        apiKey: config.sitecoreApiKey,
+        siteName: config.jssAppName,
+        tracking: true,
+      }),
+      new GraphQLLayoutFragmentService({
+        endpoint: config.graphQLEndpoint,
+        apiKey: config.sitecoreApiKey,
+        siteName: config.jssAppName,
+      })
+    );
+  }
+}
+
+export const layoutPersonalizationServiceFactory = new LayoutPersonalizationServiceFactory();
```

Comment:
I cannot find external usages of `layoutPersonalizationServiceFactory `.
`layoutPersonalizationService` seems to be used everywhere.

Reply:
factory is removed